pkg/autoscaler/multiscaler_test.go

```diff
@@ -128,6 +130,49 @@ func TestMultiScalerScaling(t *testing.T) {
 	}
 }
 
+func TestMultiScalerTickUpdate(t *testing.T) {
+	ctx := context.Background()
+	ms, stopCh, statCh, uniScaler := createMultiScaler(t)
+	defer close(stopCh)
+	defer close(statCh)
+
+	decider := newDecider()
+	decider.Spec.TickInterval = 10 * time.Second
+	uniScaler.setScaleResult(1, true)
+
+	// Before it exists, we should get a NotFound.
+	m, err := ms.Get(ctx, decider.Namespace, decider.Name)
+	if !apierrors.IsNotFound(err) {
+		t.Errorf("Get() = (%v, %v), want not found error", m, err)
+	}
+
+	_, err = ms.Create(ctx, decider)
+	if err != nil {
+		t.Errorf("Create() = %v", err)
+	}
+
+	// Expected count to be 0 as the tick interval is 10s and no autoscaling calculation should be triggered
+	if count := uniScaler.getScaleCount(); count != 0 {
```

Comment:
I think this should at least wait for a bit, say 50ms before actually saying that nothing happened.